python/feathub/processors/flink/table_builder/redis_utils.py

```diff
@@ -60,11 +61,18 @@ def insert_into_redis_sink(
         features_desc.timestamp_field,
     )
 
+    if sink.mode == RedisMode.CLUSTER:
+        redis_client: RedisCluster = RedisCluster(host=sink.host, port=sink.port)
+        servers = ",".join(f"{x.host}:{x.port}" for x in redis_client.get_nodes())
```

Comment:
Given that FeatHub python API (e.g. RedisSource) only allows users to pass a single pair of server/port, why do we need to pass a list of servers to the Java `JedisCluster`? 

I suppose Redis Cluster Client is able to identify the host/port of all servers as long as it is provided with the host/port of one server. Would it be simpler for the Java code to use instantiate `JedisCluster` using the `HostAndPort` specified by the user?